packages/f-alert/test-utils/component-objects/demo-controls.js

```diff
@@ -14,11 +14,11 @@ exports.selectAlertTypeByValue = alertType => {
 
 /**
  * Sets whether the component is dismissable based off the 'Is Dismissable' checkbox
- * @param {boolean} isDismissable
+ * @param {boolean} isDismissible
  */
-exports.isDissmiable = isDismissable => {
-    if (!isDismissable && controlIsDismissable().isSelected()) {
-        controlIsDismissable().click();
+exports.isDismisible = isDismisible => {
+    if (!isDismisible && controlIsDismissible().isSelected()) {
```

Comment:
NIT: Double s needed for consistency:
```suggestion
exports.isDismissible = isDismissible => {
    if (!isDismissible && controlIsDismissible().isSelected()) {
```